recipes/toolchain/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: toolchain
```

Comment:
Got it, still learning the wily ways of conda. Is there a canonical way we can start capturing and documenting best practices? I find the conda documentation lacking (I mean, obviously one could submit conda PRs, but I'm talking about more a cookbook for common scenarios).